src/mame/machine/6883sam.cpp

```diff
@@ -370,6 +381,54 @@ void sam6883_device::internal_write(offs_t offset, uint8_t data)
 		update_memory();
 	if (xorval & (SAM_STATE_R1|SAM_STATE_R0))
 		update_cpu_clock();
+
+	if( xorval & (SAM_STATE_F6|SAM_STATE_F5|SAM_STATE_F4|SAM_STATE_F3|SAM_STATE_F2|SAM_STATE_F1|SAM_STATE_F0))
```

Comment:
Please don’t do weird stuff with spacing around parentheses.  Follow the format of the rest of the file.